.github/ISSUE_TEMPLATE/new-story-template.yml

```diff
@@ -1,6 +1,6 @@
 name: Modernisation platform Story
 description: 📖 Create new Modernisation platform DevOps story
```

Comment:
Could we change this to...
```
description: 📖 Create Modernisation Platform story
```
I think this would read a bit better. The `new` feels redundant, and it's not exclusively going to be a DevOps story that gets raised.

Reply:
yep not a problem